hubploy/__main__.py

```diff
@@ -52,6 +52,15 @@ def main():
     deploy_parser.add_argument(
         '--version',
     )
+    deploy_parser.add_argument(
+        '--timeout',
+        nargs=1
+    )
+    deploy_parser.add_argument(
+        '--force',
+        nargs=0,
```

Comment:
You don't need this with `action='store_true'` I think - it sets the default to false & doesn't allow args to be passed to it.

Reply:
I want the default to be false, and we don't want any arguments to --force. Isn't this what we want?

Reply:
If we don't use store_true, I will need to set a default value in helm.py, which is fine.

Reply:
Yeah, as I look more carefully, it appears that if we do not use store_true, then True would need to be passed, which is not parallel to helm. Helm does not have any arguments to --force. So I think we should keep it like this.

Reply:
Ah, sorry for the confusion. I meant the `nargs=0` is not necessary with `store_true`, since it is superfluous. Hope this makes more sense.